Resources/Locale/en-US/item-recall/item-recall.ftl

```diff
@@ -0,0 +1,12 @@
+item-recall-grimoire-name = Item Recall
+item-recall-grimoire-desc = Mark a held item and summon it back at any time with just a snap of your fingers!
```

Comment:
put these into spellbook-catalog.ftl and rename it to spellbook-X-name and desc.
Eventually I'll do a refactor on the name but everything in the catalog is organized there.